scripts/stemcell-repack-debugger.sh

```diff
@@ -48,8 +52,13 @@ for arg in "$@"; do
             convert_cert_file="$2"
             shift 2
             ;;
-        --bump-version)
+        --bump-version*)
             bump_version=true
+            if [ -n "$OUT" -a -f "$OUT" ]; then
+                version=$(tar xf "$OUT" stemcell.MF --occurrence=1 -O | yq .version)
```

Comment:
Can you use bosh interpolate instead of yq here? In general I’d like to avoid adding additional dependencies